src/constants/coin-config.js

```diff
@@ -113,6 +113,10 @@ export default {
   supportedRecoveries: {
     crossChain: ['btc', 'bch', 'ltc', 'bsv'],
     nonBitGo: ['btc', 'eth', 'xrp', 'bch', 'xlm', 'ltc', 'dash', 'zec', 'btg', 'token'],
-    unsignedSweep: ['btc', 'bch', 'ltc', 'xrp', 'xlm', 'dash', 'zec', 'btg', 'eth', 'token']
+    unsignedSweep: ['btc', 'bch', 'ltc', 'xrp', 'xlm', 'dash', 'zec', 'btg', 'eth', 'token'],
+    migrated: {
+      test: ['bch', 'bsv'],
```

Comment:
should these be `tbch`, `tbsv`?

Reply:
The way the code is written it works as is. You can see none one of the other ones use the 't' prefix as all the other tools append that on in front for testnet